CMakeLists.txt

```diff
@@ -755,6 +753,16 @@ if(EXECUTORCH_BUILD_PYBIND)
                  # the torch libs are in `site-packages/torch/lib`.
                  BUILD_RPATH "@loader_path/../../../torch/lib"
                  INSTALL_RPATH "@loader_path/../../../torch/lib"
+                 BUILD_RPATH "@loader_path/../../"
```

Comment:
Please add a comment explaining these two new RPATH lines, since the "site-packages/torch-lib" comment doesn't apply to them.